test/unit_test/passes/onnx/test_conversion.py

```diff
@@ -24,3 +24,16 @@ def test_onnx_conversion_pass(input_model, tmp_path):
 
     # assert
     assert Path(onnx_model.model_path).exists()
+
+
+@pytest.mark.parametrize("target_opset", [9, 13, 16])
+def test_onnx_op_version_conversion_pass(target_opset, tmp_path):
+    input_model = get_onnx_model()
+    # setup
+    p = create_pass_from_dict(OnnxOpVersionConversion, {"target_opset": target_opset}, disable_search=True)
+    output_folder = str(tmp_path / "onnx")
+
+    onnx_model = p.run(input_model, None, output_folder)
+
+    # assert
+    assert Path(onnx_model.model_path).exists()
```

Comment:
could also load the model and verify the opset version?

Reply:
It should be. Updated.